pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/db/dto/CancerItem.java

```diff
@@ -0,0 +1,60 @@
+package org.broadinstitute.ddp.db.dto;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * The name of a cancer and its language
+ */
+public class CancerItem {
```

Comment:
The cancer list was just a bunch of strings, so we needed a new class that could represent both the cancer name and the language.